src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocvariable/InputJavadocVariableInner2.java

```diff
@@ -0,0 +1,84 @@
+// violation
```

Comment:
Why violation here? I don't see any violations in this test
https://github.com/checkstyle/checkstyle/blob/dabdffd3716c6fdbb780faff63e26ceb08d2bbe3/src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocVariableCheckTest.java#L91-L99

Reply:
I have same question , but if i try to remove this  it gives error on ```mvn clean verify```
error massage:
```
[ERROR] [checkstyle] [ERROR] H:\GSOC\checkstyle\src\test\resources\com\puppycrawl\tools\checkstyle\checks\javadoc\javadocvariable\InputJavadocVariableInner2.java:0: All check input files in the 'src/(it|test)/resources*/' folder must                    have trailing comments "// ok" and/or "// violation". [StyleValidationCommentInInputs]
```

Reply:
so if i replace ``` // validation ``` with ```// ok``` is that fine?

Reply:
No. At top of file, there must only be package declaration. There are so many `ok` cases below. Just comment in some (or one) of the,/

Reply:
ok

Reply:
Resolved.